_data/guides/quelle-api-sirene.md

```diff
@@ -79,27 +82,27 @@ En revanche certaines données ne sont pas présentes dans toutes les API :
 
 Le tableau ci-dessous fait une synthèse de ces différence pour vous aider a choisir :
 
-| API                                     | API Sirene données ouvertes | API Sirene de l'INSEE | API Entreprise |
-| --------------------------------------- | --------------------------- | --------------------- | -------------- |
-| Etablissements diffusibles              | ✅ Oui                      | ✅ Oui                | ✅ Oui         |
-| Etablissements non-diffusibles          | ❌ Non                      | ✅ Oui                | ✅ Oui         |
-| Refus d'immatriculation par le Greffe   | ❌ Non                      | ✅ Oui                | ❌ Non         |
-| Recherche des prédecesseurs/successeurs | ❌ Non                      | ✅ Oui                | ❌ Non         |
-| geolocalisation                         | ✅ Oui                      | ❌ Non                | ❌ Non         |
+| API                                     | API Sirene de l'INSEE | API Entreprise |
+| --------------------------------------- | --------------------- | -------------- |
+| Etablissements diffusibles              | ✅ Oui                | ✅ Oui         |
+| Etablissements non-diffusibles          | ✅ Oui                | ✅ Oui         |
+| Refus d'immatriculation par le Greffe   | ✅ Oui                | ❌ Non         |
+| Recherche des prédecesseurs/successeurs | ✅ Oui                | ❌ Non         |
+| geolocalisation                         | ❌ Non                | ❌ Non         |
+| recherche par dénomination              | ✅ Oui                | ❌ Non         |
```

Comment:
So from what I understand `API recherche d'entreprises` is not be included in this comparison?